implementations/rust/ockam/ockam_command/src/command_global_opts.rs

```diff
@@ -139,10 +139,6 @@ impl CommandGlobalOpts {
         cmd: &OckamSubcommand,
         is_tty: bool,
     ) -> miette::Result<LoggingConfiguration> {
-        if global_args.quiet {
```

Comment:
I think we should write a good description, and possibly some bats tests for the interactions between `quiet`, `-vvv`, `export OCKAM_LOGGING=true`.

Reply:
The current [description](https://github.com/build-trust/ockam/blob/33160ea84a6bbf4e0c4d162cb20a5ca99f192a95/implementations/rust/ockam/ockam_command/src/global_args.rs#L21) is pretty accurate in my opinion. The `quiet` flag should only control whether we are allowed to write to stderr.

Would you add something else?

Reply:
Actually I would:

 - Enforce and specify the following priorities
    - file configuration variable < env. variable < command line parameter
 - Use some naming conventions:
    - option in configuration file: `an_option`
    - same option as an environment variable:  `OCKAM_AN_OPTION`
    - same option as a command-line option: `an-option`

That means that:

 - We introduce a `--logging` option corresponding to `OCKAM_LOGGING`, set to `false` by default.
 - We introduce a `--confirm` option set to `true` by default.
 - We specify that `--quiet` is a flag meaning `--logging false --confirm false` as a shortcut.
 - We similarly explain that `-vvv` is a flag meaning `--logging true --log-level info` as a shortcut.

Reply:
I like the idea of adding the `logging` and `log-level` flags. But note that the `quiet` has its own meaning: disable interactivity + displaying less output, and it doesn't have anything to do with logging, it's only CLI-related.

Reply:
Ah, ok. that was not clear to me from the description: `Do not print any log messages` :-).